leasing/models/area.py

```diff
@@ -5,7 +5,7 @@
 
 class Area(TimestampedModelMixin):
     name = models.CharField(max_length=255)
-    mpoly = models.MultiPolygonField(srid=3857)
+    mpoly = models.MultiPolygonField(srid=4326)
```

Comment:
It would be good to name the field according to its function, not its type. `geometry` would be a good choice.